.github/workflows/registry_upload.yml

```diff
@@ -7,6 +7,9 @@ on:
     branches: [main]
     types:
       - completed
```

Comment:
Should we drop the `workflow_run` bit? I think you would expect `latest` to point to the latest `release`, not just the latest commit?

Reply:
Yes, we can drop the workflow_run.